content/cli/get-started/examples/leaderboard.md

```diff
@@ -166,10 +166,10 @@ content:
 Create this dataset:
 
 ```sh
-kamu add player-scores.yaml
+kamu add leaderboard.yaml
 ```
 
-Let's process all data that is already in `player-scores` and see the results:
+Let's process all data that is already in `leaderboard` and see the results:
```

Comment:
I think the original version of this line is correct. Although we are pulling the `leaderboard` dataset, it uses `payer-scores` as an input, therefore we:
```
... process all data that is already in `player-scores`
```

I can see how this can be confusing tough, so perhaps we should change this line to:
```
Let's "pull" the `leaderboard` dataset to process all the data that is already in `player-scores` and check the results:
```
to remove any ambiguity?